### PR TITLE
new IceCube notice types

### PR DESCRIPTION
**Pull request by volodymyrss**
Looks like there are new notice types:

171  HAWC_BURST_MONITOR 
173  ICECUBE_ASTROTRACK_GOLD   
174  ICECUBE_ASTROTRACK_BRONZE

https://gcn.gsfc.nasa.gov/amon.html
https://arxiv.org/pdf/1903.08714.pdf

### CODE REVIEW

File: gcn/notice_types.py [hunk 1]
```diff
@@ -154,7 +154,10 @@
     CALET_GBM_FLT_LC=160,
     CALET_GBM_GND_LC=161,
     LVC_RETRACTION=164,
-    AMON_ICECUBE_EHE=169)
+    AMON_ICECUBE_EHE=169,
+    HAWC_BURST_MONITOR=171,
+    ICECUBE_ASTROTRACK_GOLD=173,
```
[the remark this answers is not in the document]
Comment (volodymyrss) [reply gcn/notice_types.py]: they are also called "in-work" there while in fact they should be almost certainly considered ACTIVE: a number of them were sent. 
clearly the "filtering" page is not up to date.

though I understand it makes more sense to use a single reference.
I contact Scott.

[the remark this answers is not in the document]
Comment (lpsinger) [reply gcn/notice_types.py]: Any information yet?

[the remark this answers is not in the document]
Comment (volodymyrss) [reply gcn/notice_types.py]: I was traveling and it seems like my mail to Scott maybe was not delivered? did you get a CC 5 days ago?
I sent another one right now.
Thanks for the reminder!

[the remark this answers is not in the document]
Comment (volodymyrss) [reply gcn/notice_types.py]: the page is updated now:

https://gcn.gsfc.nasa.gov/filtering.html

Num | TYPE MACRO_NAME | STATUS | Ext vs Int | COMMENTS
-- | -- | -- | -- | -- 
171 | HAWC_BURST_MONITOR | ACTIVE | EXTERNAL | HAWC Instrument High Energy Gamma-ray 
173 | ICECUBE_ASTROTRACK_GOLD | ACTIVE | EXTERNAL |   ICECUBE High Energy Single (neutrino) Event 
174 | ICECUBE_ASTROTRACK_BRONZE | ACTIVE | EXTERNAL |   ICECUBE High Energy Single (neutrino) Event (less confident real)

this matches what is already in this PR

File: gcn/notice_types.py [hunk 2]
```diff
@@ -154,7 +154,10 @@
     CALET_GBM_FLT_LC=160,
     CALET_GBM_GND_LC=161,
     LVC_RETRACTION=164,
-    AMON_ICECUBE_EHE=169)
+    AMON_ICECUBE_EHE=169,
+    HAWC_BURST_MONITOR=171,
```
[the remark this answers is not in the document]
Comment (lpsinger) [reply gcn/notice_types.py]: Thank you!

**Pull request merged**